Script/AtomicEditor/ui/frames/MainFrame.ts

```diff
@@ -111,12 +111,18 @@ class MainFrame extends ScriptWidget {
         if (this.menu.handlePopupMenu(target, refid))
             return true;
 
+        if (this.openedMenu && this.openedMenu.id == target.id + " popup") {
+            this.openedMenu.close();
+            this.openedMenu = null;
+            return true;
+        }
+
```

Comment:
close() removes the native pointer, so openedMenu should be GCed,
don't really need set it to null